tests/util/test_traceutils.py

```diff
@@ -103,17 +104,28 @@ def test_is_service_or_endpoint_ignored() -> None:
     agent.options.ignore_endpoints.append("service2.endpoint1")
 
     # ignore all endpoints of service1
-    assert is_service_or_endpoint_ignored("service1")
-    assert is_service_or_endpoint_ignored("service1", "endpoint1")
-    assert is_service_or_endpoint_ignored("service1", "endpoint2")
+    assert agent._HostAgent__is_service_or_endpoint_ignored("service1")
+    assert agent._HostAgent__is_service_or_endpoint_ignored("service1", "endpoint1")
+    assert agent._HostAgent__is_service_or_endpoint_ignored("service1", "endpoint2")
 
     # case-insensitive
-    assert is_service_or_endpoint_ignored("SERVICE1")
-    assert is_service_or_endpoint_ignored("service1", "ENDPOINT1")
+    assert agent._HostAgent__is_service_or_endpoint_ignored("SERVICE1")
+    assert agent._HostAgent__is_service_or_endpoint_ignored("service1", "ENDPOINT1")
 
     # ignore only endpoint1 of service2
-    assert is_service_or_endpoint_ignored("service2", "endpoint1")
-    assert not is_service_or_endpoint_ignored("service2", "endpoint2")
+    assert agent._HostAgent__is_service_or_endpoint_ignored("service2", "endpoint1")
+    assert not agent._HostAgent__is_service_or_endpoint_ignored("service2", "endpoint2")
 
     # don't ignore other services
-    assert not is_service_or_endpoint_ignored("service3")
+    assert not agent._HostAgent__is_service_or_endpoint_ignored("service3")
+
+
+@pytest.mark.parametrize(
+    "span_name, expected_result",
+    [("something", ""), ("redis", "command"), ("dynamodb", "op")],
+)
+def test_get_operation_specifier(
+    span_name: str, expected_result: Optional[str]
+) -> None:
+    response_redis = get_operation_specifier(span_name)
+    assert response_redis == expected_result
```

Comment:
If this function is placed in the `span_utils.py` file, it is placed wrongly here. 
Please create a new test file for the new file created.